src/Symfony/Component/RateLimiter/NoLimiter.php

```diff
@@ -23,9 +23,9 @@
  */
 final class NoLimiter implements LimiterInterface
 {
-    public function consume(int $tokens = 1): bool
+    public function consume(int $tokens = 1): Limit
     {
-        return true;
+        return new Limit(\INF, new \DateTimeImmutable(), true, 'no_limit');
```

Comment:
```suggestion
        return new Limit(\INF, new \DateTimeImmutable(), true, static::class);
```

Reply:
FQCN is now removed, so I will use string key again and set it in constructor but nice idea !